src/components/PeoplePage/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from 'react';
+
+import classNames from 'classnames';
+
+import { useParams } from 'react-router-dom';
+import { PeopleLink } from '../PeopleLink/PeopleLink';
+
+import { Person } from '../../../types';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  function getFindPerson(name: string) {
+    return people.find(currentPerson => currentPerson.name === name);
+  }
+
+  useEffect(() => {
+    const selectedPerson = document.querySelector('.has-background-warning');
+
+    if (selectedPerson) {
+      selectedPerson.scrollIntoView({
+        block: 'center',
+        behavior: 'smooth',
+      });
+    } else {
+      window.scrollTo({
+        top: 0,
+        behavior: 'smooth',
+      });
+    }
+  }, [slug]);
+
+  const getParent = (
+    parent: Person | undefined, personParent: string,
+  ) => {
+    return parent
+      ? <PeopleLink person={parent} />
+      : personParent;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = person.motherName
+            ? getFindPerson(person.motherName)
+            : undefined;
+          const father = person.fatherName
+            ? getFindPerson(person.fatherName)
+            : undefined;
+
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable outside the template 
please check other places